chpl/chpl-api/src/main/java/gov/healthit/chpl/questionableActivity/CertificationResultQuestionableActivityProvider.java

```diff
@@ -177,31 +222,39 @@ public QuestionableActivityCertificationResultDTO checkGapUpdated(
         }
         return addedMacras;
     }
-    
-    public List<QuestionableActivityCertificationResultDTO> checkG2MacraMeasuresRemoved(
-            CertificationResult origCertResult, CertificationResult newCertResult) {
-        
-        List<QuestionableActivityCertificationResultDTO> removedMacras = new ArrayList<QuestionableActivityCertificationResultDTO>();
-        if ((newCertResult.getG2MacraMeasures() == null || newCertResult.getG2MacraMeasures().size() == 0) && 
-                origCertResult.getG2MacraMeasures() != null && origCertResult.getG2MacraMeasures().size() > 0) {
+
+      /**
+       * Check for QA re: removal of G2 MACRA measures.
+       * @param origCertResult original certification criteria result
+       * @param newCertResult new certification criteria result
+       * @return list of removed measures
+       */
+       public List<QuestionableActivityCertificationResultDTO> checkG2MacraMeasuresRemoved(
+            final CertificationResult origCertResult, final CertificationResult newCertResult) {
+
+        List<QuestionableActivityCertificationResultDTO> removedMacras
+        = new ArrayList<QuestionableActivityCertificationResultDTO>();
```

Comment:
Should this be indented?  https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent